chall08/szavarce.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python
+import sys
+import re
+
+class Position:
```

Comment:
This is personal opinion, but I think the class should represent an Object, than has a position himself, like representing the robot, or a mine, etc. But good job on using classes.